networkx/algorithms/bridges.py

```diff
@@ -68,20 +70,33 @@ def all_local_bridges(G, first_match=False):
                 G.add_edge(startNode, endNode)
                 if pathLength > 2:
                     # found a local bridge
-                    allBridges.append([startNode, endNode])
-                    allSpans.append(pathLength)
+                    bridgeObjs.append(
+                        {"edge": [startNode, endNode], "span": pathLength})
                     if first_match:
+                        allBridges.append(sorted([startNode, endNode]))
+                        allSpans.append(pathLength)
                         return {"edges": allBridges, "spans": allSpans}
 
             except nx.NetworkXNoPath:
                 # found a bridge
-                allBridges.append([startNode, endNode])
-                allSpans.append(-1)
+                bridgeObjs.append({"edge": [startNode, endNode], "span": -1})
                 G.add_edge(startNode, endNode)
                 if first_match:
+                    allBridges.append(sorted([startNode, endNode]))
+                    allSpans.append(-1)
                     return {"edges": allBridges, "spans": allSpans}
         usedNodes.append(startNode)
 
+    # sort the list in ascending order (for consistency in output)
+    for bridge in bridgeObjs:
```

Comment:
I did the sort because my original code sometimes gave different orderings of edges and for consistency with the doctest in Travis CI, I thought it would be necessary to make the order of output edges exactly match. Running your code suggestion for at least the local bridges function seems to have solved that issue as well. So yes, the ordering isn't needed anymore.